container/worker/run_openqa_worker.sh

```diff
@@ -6,4 +6,16 @@ if [[ -z $OPENQA_WORKER_INSTANCE ]]; then
 fi
 
 /root/qemu/kvm-mknod.sh
+
+if [ -e "/dev/kvm" ]; then
+  if [ ! $(getent group kvm) ]; then
```

Comment:
This should be much simpler since `getent` seems to return `0` only if the group exists:
```suggestion
  if getent group kvm > /dev/null; then
```

My version also negates the condition so `groupmod` is only executed when the group exists. I suppose that's the intention here (because the `groupmod` command would simply fail otherwise).

Reply:
The problem here is that /dev/kvm could has groups that are not in the system (and with an ls you can se a number) So the objective with groupmod is to create the kvm group using the current group. So only if the group kvm doesn't exist should be executed groupmod

Reply:
But the `groupmod` command does *not* create a new group, e.g.:

```
LANG=C sudo /usr/sbin/groupmod -g 123 foo
groupmod: group 'foo' does not exist
```

As I said, it only alters the group ID of the existing group (in this case to match the group ID of `/dev/kvm`).

(The command to add a group is `groupadd` btw.)

Reply:
Just to be on the same page: Your `groupmod` command fixes a mismatch between the underlying group ID of the group `kvm` in and outside the container. It adjusts the kvm group within the container to have the same group ID as the kvm group outside the container which fixes the permission errors because the device mapped into the container still uses the group ID from outside the container. And again, the command does *not* create a group which is missing within the container (although this might be nice to have as well as mentioned in the other comments).

Reply:
Thanks for the detailed explanation